.env.integration

```
@@ -4,6 +4,26 @@ APP_PUBLIC_URL=https://integration.passculture.app
 # Feature Toggling
 FEATURE_FLIPPING_ONLY_VISIBLE_ON_TESTING=false
 
+APP_DISPLAY_NAME = 'pass Culture I'
+
+# iOS
+IOS_APP_ID='app.passculture.integration'
+IOS_APP_NAME='PassCulture I'
+IOS_PROVISIONING_PROFILE_SPECIFIER_DEVELOPMENT='match Development app.passculture'
```

si c'était pas défini, j'imagine que ce n'était pas utilisé et donc pas utile ?

est-ce qu'il faudrait pas faire

```suggestion
IOS_APP_ID="NOT USED IN INTEGRATION ENVIRONMENT"
IOS_APP_NAME="NOT USED IN INTEGRATION ENVIRONMENT"
IOS_PROVISIONING_PROFILE_SPECIFIER_DEVELOPMENT="NOT USED IN INTEGRATION ENVIRONMENT"
```